docs/can-canjs/can-infrastructure.md

```diff
@@ -50,8 +63,7 @@ The following makes the `Person` type’s `getName()` observable:
 
 ```js
 import Observation from 'can-observation';
-import canEvent from 'can-event';
-import assign from 'can-util/js/assign/assign';
+import mixinMapBindings from 'can-event-queues/map/map';
```

Comment:
Does this need to be mixed in for the code below to work?

Reply:
Updated!